vertical-pod-autoscaler/pkg/recommender/routines/recommender.go

```diff
@@ -80,6 +78,7 @@ type recommender struct {
 	podResourceRecommender        logic.PodResourceRecommender
 	useCheckpoints                bool
 	lastAggregateContainerStateGC time.Time
+	recommendationPostProcessor   []RecommendationPostProcessor
```

Comment:
```suggestion
	recommendationPostProcessors   []RecommendationPostProcessor
```
This should probably be plural?